MLDSAI/OpenAdapt
# Automatic Publishing with PyPi

## Overview

- Resolves #22 
- Using Github Actions script that automatically publishes when a new release is created
- TODO: use automatic semantic versioning with PyPi

## Unit Tests Created

- n/a

## Steps to QA

- See Github Actions run history on my repo: https://github.com/Mustaballer/puterbot/actions
- See new Release on my forked repo: https://github.com/Mustaballer/puterbot
- Check out: https://test.pypi.org/project/openadapt/
